gcloud/streaming/http_wrapper.py

```diff
@@ -45,9 +61,9 @@
 # http_request: A http_wrapper.Request.
 # exc: Exception being raised.
 # num_retries: Number of retries consumed; used for exponential backoff.
```

Comment:
Can you turn this into a docstring on the constant? (I know it's non-public, but a docstring is better than a comment passing itself off as one.)